### PR TITLE
Radio Button - box border should change from 60% to 100% both on hover and focus

### PR DESCRIPTION
**Pull request by urig-wix**
# ✨ Pull Request

### 💁 Description
The box border should change from 60% to 100% both on hover and focus.
See comment on [EE-28048](https://jira.wixpress.com/browse/EE-28048?focusedCommentId=8601528&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel)

...

### 💭 Motivation

Would like to migrate to radio button at our project.
...

### ☝️ TODOs 

...

### 👀  PR is ready for review 

- [x] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_638/

File: src/components/RadioButton/RadioButton.st.css
Comment (jonathanadler) [inline src/components/RadioButton/RadioButton.st.css:203]: I'm not sure this is needed, I think that it's always defined without opacity when it's `checked`

File: src/components/RadioButton/RadioButton.st.css
Comment (jonathanadler) [inline src/components/RadioButton/RadioButton.st.css:207]: And then maybe we don't need the `:not(:checked)` her.
Maybe

File: src/components/RadioButton/RadioButton.st.css
Comment (jonathanadler) [inline src/components/RadioButton/RadioButton.st.css:210]: should this be:
```suggestion
    border-color: applyOpacity(value(CurrentBorderColor), 1);
```
?

**Review by jonathanadler — COMMENTED** [review]
Looks good as always 👍 
left some comments and questions.

Also unfortunately we don't have a good way to test this at the moment :( .

File: src/components/RadioButton/RadioButton.st.css
Comment (urig-wix) [reply src/components/RadioButton/RadioButton.st.css:207]: without the :not(:checked), the color of the border would change when hovering (gets 'CurrentIconColor' instead of 'CurrentBorderColor')

File: src/components/RadioButton/RadioButton.st.css
Comment (urig-wix) [reply src/components/RadioButton/RadioButton.st.css:210]: deleted ^ as its included in 
.root:hover .radioIcon {
    border-color: applyOpacity(value(CurrentBorderColor), 1);
}
(and the box does not have opacity at all when it's checked)

**Review by jonathanadler — APPROVED** [review]
Awesome 😎

Can you send the storybook link for johanna to review?

I'm approving, but we'll just wait for her approval before merging

**Comment by urig-wix** [comment]
> Awesome 😎
> 
> Can you send the storybook link for johanna to review?
> 
> I'm approving, but we'll just wait for her approval before merging

Sorry for the newbies question but how do I send a link of the branches storybook ?

**Comment by jonathanadler** [comment]
> > Awesome 😎
> > Can you send the storybook link for johanna to review?
> > I'm approving, but we'll just wait for her approval before merging
> 
> @jonathanadler Sorry for the newbies question but how do I send a link of the branches storybook ?

No need to apologise 😊 

You can find it if you scroll up a bit
image

File: src/components/RadioButton/RadioButton.st.css
Comment (jonathanadler) [inline src/components/RadioButton/RadioButton.st.css:215]: Note that this will add the transition only on hover, but not when removing the cursor/focus from the element.
Is that ok?

Comment (urig-wix) [reply src/components/RadioButton/RadioButton.st.css:215]: that's a good question. 
should I ask Johanna? what would you do different?

Comment (jonathanadler) [reply src/components/RadioButton/RadioButton.st.css:215]: Well, if we add it directly to the `radioIcon`, like this:
```css
.radioIcon {
  transition: border-color .2s ease-in-out;
}
```
It will affect both the enter and the leave.
However, this will also affect the border in all the radio-button's states: when the radio-button is being checked, disabled, etc.
so I guess it's best to leave it like this.
I would check with Johanna in any case, though I'm not sure achieving this is worth the effort.
(and I raised the issue in the first place!)

**Pull request merged**
